bittensor_cli/src/bittensor/utils.py

```diff
@@ -558,7 +558,11 @@ def format_error_message(error_message: Union[dict, Exception]) -> str:
             err_type = error_message.get("type", err_type)
             err_name = error_message.get("name", err_name)
             err_docs = error_message.get("docs", [err_description])
-            err_description = err_docs[0] if err_docs else err_description
+            if err_docs:
+                if isinstance(err_docs, list):
+                    err_description = err_docs[0]
```

Comment:
```suggestion
                    err_description =  " ".join(err_docs)
```
theoretically, a docs can consist of few elements containing a description.
```py
err_docs = ["This", "is", "a few", "item", "description."]
print(" ".join(err_docs))
>>> 'This is a few item description.'
```